readme.md

```diff
@@ -47,7 +47,7 @@ go test -cover ./...
 
 then through Telnet or any TCP connection
 ```
-telnet 127.0.0.1 8080
+openssl s_client -starttls smtp -connect 127.0.0.1:8080
```

Comment:
we should document based on `redis-cli`
this document can give u more insights 
https://redislabs.com/blog/stunnel-secure-redis-ssl/